.travis.yml

```diff
@@ -7,6 +7,16 @@ services:
 # a lightweight environment
 language: c
 
+env:
+    - DC=dmd1 DIST=trusty F=production
+    - DC=dmd1 DIST=trusty F=devel
+    - DC=dmd1 DIST=xenial F=production
+    - DC=dmd1 DIST=xenial F=devel
+    - DC=dmd-transitional DIST=trusty F=production
+    - DC=dmd-transitional DIST=trusty F=devel
+    - DC=dmd-transitional DIST=xenial F=production
+    - DC=dmd-transitional DIST=xenial F=devel
```

Comment:
I was envisioning simply using docker tags for base images that depend on compiler version and pretty much keeping existing scripts as they are apart from that. it is more of a matter of getting more readable test report matrix in travis than actual infrastructure.